--- movit-1.5.2/Makefile.in.orig	2017-08-09 22:37:21.139230303 +0200
+++ movit-1.5.2/Makefile.in	2017-08-09 22:37:36.344443738 +0200
@@ -93,7 +93,7 @@ TESTS=effect_chain_test fp16_test $(TEST
 LIB_OBJS=effect_util.o util.o effect.o effect_chain.o init.o resource_pool.o ycbcr.o $(INPUTS:=.o) $(EFFECTS:=.o)
 
 # Default target:
-all: libmovit.la $(TESTS)
+all: libmovit.la
 
 ifeq ($(with_demo_app),yes)
 all: demo
